lib/recipients/project.js

```diff
@@ -117,7 +115,8 @@ module.exports = async ({ schema, logger, task }) => {
       emailTemplate: 'project-expired',
       addTaskTypeToSubject: false,
       applicant: project.licenceHolder,
-      logMsg: 'project has expired'
+      logMsg: 'project has expired',
+      identifier: `${project.id}-${action}`
```

Comment:
```suggestion
      identifier: `${project.id}-expired`
```

Depending on what `action` is, this might be blocked by the previous "expiring" messages.

Reply:
The actions have the months in them: 
```
'project-expiring-12',    'project-expiring-6',    'project-expiring-3',    'project-expired'
```